release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -557,6 +557,76 @@ For more information, see xref:../networking/metallb/about-metallb.adoc#about-me
 
+[id="ocp-4-10-storage-alicloud-disk-csi-operator"]
+==== Persistent storage using the Alibaba AliCloud Disk CSI Driver Operator
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for AliCloud Disk. The AliCloud Disk Driver Operator that manages this driver is generally available.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-alicloud-disk.adoc#persistent-storage-csi-alicloud-disk[AliCloud Disk CSI Driver Operator].
+
+[id="ocp-4-10-storage-azure-file-csi-operator"]
+==== Persistent storage using the Microsoft Azure File CSI Driver Operator (Technology Preview)
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Azure File. The Azure File Driver Operator that manages this driver is in Technology Preview.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-azure-file.adoc#persistent-storage-csi-azure-file[Azure File CSI Driver Operator].
+
+[id="ocp-4-10-storage-ibm-vpc-block-csi-operator"]
+==== Persistent storage using the IBM VPC Block CSI Driver Operator
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for IBM Virtual Private Cloud (VPC) Block. The IBM VPC Block Driver Operator that manages this driver is generally available.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-ibm-vpc-block.adoc#persistent-storage-csi-ibm-vpc-block[IBM VPC Block CSI Driver Operator].
+
+[id="ocp-4-10-storage-vsphere-csi-operator"]
+==== Persistent storage using VMware vSphere CSI Driver Operator is generally available
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for vSphere. This feature was previously introduced as a Technology Preview feature in {product-title} 4.8 and is now generally available and enabled by default in {product-title} 4.10.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere[vSphere CSI Driver Operator].
+
+vSphere CSI Driver Operator installation requires:
+
+* Certain minimum component versions installed. See xref:../release_notes/ocp-4-10-release-notes.adoc#ocp-4-10-installation-vsphere-csi[CSI driver installation on vSphere clusters]
+
+* Removal of any non-Red Hat vSphere CSI driver (xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere-install-issues_persistent-storage-csi-vsphere[Removing a non-Red Hat vSphere CSI Operator Driver])
```

Comment:
@lpettyjo This link did not mention anything about how to remove non-Red Hat vSphere CSI driver or related info, could we remove this link? 
xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere-install-issues_persistent-storage-csi-vsphere

Reply:
@duanwei33 I'm not sure I understand this comment. That link does provide info about the implications of having a non-RH driver and a brief procedure on how to remove them:  

"To uninstall the non-Red Hat vSphere CSI Driver:

Delete the non-Red Hat vSphere CSI Driver (VMware vSphere Container Storage Plug-in) Deployment and Daemonset objects.

Delete the configmap and secret objects that were installed previously with the non-Red Hat vSphere CSI Driver.

Delete the non-Red Hat vSphere CSI driver CSIDriver object:"

Reply:
@lpettyjo Oh my fault, looks like I got the wrong link last time, yes, this link does provide the correct info, really sorry for that.